### PR TITLE
Create.ini

### PR DESCRIPTION
**Pull request by anyjser**

### CODE REVIEW

File: files/nviewjs/update.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "packageManager": "github",
+  "name": "nviewjs",
+  "repo": "any-js/nviewjs",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "nview.css", "nview.min.css",
+      "nview.js", "nview.min.js"
```
Comment (jimaek) [inline files/nviewjs/update.json]: Can you just leave the min files?
You can use this rule `*.min.*` to add them all

File: files/nviewjs/update.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "packageManager": "github",
+  "name": "nviewjs",
+  "repo": "any-js/nviewjs",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "nview.css",
+      "nview.js",
+      "*.min.*"
```
Comment (jimaek) [inline files/nviewjs/update.json]: Thats not what I meant. Remove all files and just leave `*.min.*` there

**Comment by anyjser** [comment]
Sorry, I misunderstood. I modified json-file.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**